src/main/java/pl/matidominati/GitHubApp/client/model/GitHubRepo.java

```diff
@@ -0,0 +1,26 @@
+package pl.matidominati.GitHubApp.client.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class GitHubRepo {
```

Comment:
Moznaby sie zastanowic czy potzrebujemy tych wszystkich pól do realizacji naszego zadania -> nie jest to błąd tylko preferencja